auth/src/main/java/com/redhat/lightblue/rest/auth/jboss/CertLdapLoginModule.java

```diff
@@ -127,7 +130,18 @@ public void initializeLightblueLdapRoleProvider() throws NamingException {
             ldapConf.poolMaxConnectionAgeMS(Integer.parseInt((String)options.get(POOL_MAX_CONNECTION_AGE_MS)));
         }
 
-        lbLdap = new LightblueLdapRoleProvider(searchBase, ldapConf);
+        int rolesCacheExpiry = 5*60*1000; // default 5 minutes
+        if (options.containsKey(ROLES_CACHE_EXPIRY_MS)) {
```

Comment:
It's decoupled from puppet changes this way. We don't need to update puppet unless we want to change the optional values.